tests/microos/disk_boot.pm

```diff
@@ -23,6 +23,7 @@ sub run {
     # SLEM updated GM images from https://openqa.suse.de/group_overview/377
     # already have disabled grub timeout in order to install updates and reboot
     # therefore *aarch64* images would hang in GRUB2
+    sleep;
```

Comment:
I guess this PR is in troubleshooting mode :)

Reply:
What is the issue, btw?

Reply:
sorry, I have missed the mail notification, https://bugzilla.suse.com/show_bug.cgi?id=1212781